src/stcal/alignment/util.py

```diff
@@ -323,7 +327,7 @@ def wcs_from_footprints(
     ref_wcs: gwcs.wcs.WCS,
     ref_wcsinfo: dict,
     transform: astropy.modeling.models.Model | None = None,
-    bounding_box: Sequence | None = None,
+    bounding_box: Sequence | None = None,  # noqa: ARG001, ARG002
```

Comment:
Follow up issue: https://github.com/spacetelescope/stcal/issues/464 we should probably just remove this function.